### ISSUE TITLE
Comments Menu Item shows <span> for Comments Awaiting Moderation

### ISSUE THREAD

**Issue by d4mation** [joelworsham/cd-new-temp#2]
Maybe a special case for Comments should be made so that the User cannot remove this `<span>`? For some users seeing that as part of the Menu Item "Name" could be confusing and it has functional use within WP Core.

![image](https://cloud.githubusercontent.com/assets/7770631/23512327/3c6c4a42-ff2e-11e6-8eed-b98023cec54d.png)

**Comment by d4mation** [comment]
Similarly when Managing the Dashboard and you view the "Quick Draft" button.

Although this one technically has _two_ editable Strings: for JS and one for no JS.

![image](https://cloud.githubusercontent.com/assets/7770631/23513059/c318988c-ff30-11e6-8582-fee531e8e504.png)

**Comment by joelworsham** [comment]
Also plugins when there's updates. This is a more broad issue of "you are allowed to process HTML within menu items and dashboard titles, but how do we handle that within customizing?".

As much as it looks... well not great... it *does* allow the user to add HTML to the titles, which is kind of neat. I'm not really sure what a good alternative here would be besides completely stripping out the HTML in the first place, but then you lose the functionality of it.

**Comment by d4mation** [comment]
Yeah, it is difficult.

For Plugins/Comments, maybe it could always tack on that `<span>` to a custom String? That way they can still add their own HTML if they really want to, but they don't lose the little indicator?

The "Quick Draft" link would be more complicated though.

**Comment by d4mation** [comment]
But that's assuming the indicator works via AJAX. It probably doesn't :/ 

You'd need to grab the value somewhere via PHP and populate it.

**Comment by joelworsham** [comment]
Right. I can try to manually cover a few bases here manually, but there can, and probably will, always be more popping up. I believe WooCommerce also utilizes this for Orders.

Really tough call here. Let's keep this in our minds for a while and figure something out.
